src/api/Groups.js

```diff
@@ -72,33 +53,6 @@ class Groups extends Base {
             }),
         );
     }
-
-    /**
-     * API for fetching all groups in the current user's enterprise
-     *
-     * @param {string} id - Box item ID
-     * @param {Function} successCallback - Success callback
-     * @param {Function} errorCallback - Error callback
-     * @param {string} [filterTerm] - Optional filter for the groups
-     * @param {Object} [requestData] - Opitional additional request data
-     * @returns {void}
-     */
-    getGroupsInEnterprise(
-        id: string,
-        successCallback: Function,
-        errorCallback: ElementsErrorCallback,
-        filterTerm: ?string,
-        requestData: ?Object,
-    ): void {
-        this.errorCode = ERROR_CODE_FETCH_ENTERPRISE_GROUPS;
-        this.get({
-            id,
-            successCallback,
-            errorCallback,
-            url: this.getGroupsInEnterpriseUrl(filterTerm),
-            requestData,
-        });
-    }
```

Comment:
Did groups and users APIs change? Can we still have offset based pagination for these endpoints?

Reply:
I added these functions, so they're not currently being used for anything else in BUIE. I can add them back in, though. Thoughts?

Reply:
@sahiga, if you are removing the same functions that you added recently, then it's ok.